src/services/projects.js

```diff
@@ -420,14 +421,19 @@ module.exports = class ProjectsHelper {
 	 * @returns {JSON} - List of reviewers from the org
 	 */
 
-	static async reviewerList(organization_id, pageNo, limit) {
+	static async reviewerList(user_id, organization_id, pageNo, limit) {
 		try {
-			const reviewers = await userRequests.list(common.REVIEWER, pageNo, limit, '', organization_id)
+			let reviewers = await userRequests.list(common.REVIEWER, pageNo, limit, '', organization_id)
+			const userList = reviewers.data.result.data.filter((user) => user.id !== user_id)
+
 			if (reviewers.success) {
```

Comment:
@adithyadinesh0412  if reviewers is success and data is there the only do the removal